app/controllers/specialist_document_controller.rb

```diff
@@ -1,5 +1,7 @@
 class SpecialistDocumentController < ContentItemsController
   include Cacheable
 
-  def show; end
+  def show
+    @specialist_document_presenter = SpecialistDocumentPresenter.new(content_item)
```

Comment:
:+1:

(a bunch of existing examples of this code have content_item.to_h, we should probably clean them up)